test/parallel/test-icu-stringwidth.js

```diff
@@ -1,15 +1,15 @@
 // Flags: --expose_internals
 'use strict';
-
 const common = require('../common');
-const assert = require('assert');
-const readline = require('internal/readline');
```

Comment:
@mscdex If I'm not wrong, we are doing the same for `crypto`, where we require the additional modules only if `common.hasCrypto` is true.